applications/luci-app-openvpn/luasrc/model/cbi/openvpn-advanced.lua

```diff
@@ -371,6 +371,10 @@ local knownParams = {
 			{ "net30", "p2p", "subnet" },
 			translate("'net30', 'p2p', or 'subnet'"),
 			{dev_type="tun" } },
+		{ Flag,
+			"disable_dco",
+			0,
+			translate("Disable data channel offloading support") },
```

Comment:
revise to the following to help users searching settings based on acronyms:
```lua
			translate("Disable Data Channel Offloading (DCO) support") },
```

Reply:
OK, I'll change that.